partitura/utils/music.py

```diff
@@ -958,23 +1071,31 @@ def pianoroll_to_notearray(pianoroll,
             notes[-1][3].append(pianoroll[n, at])
 
     # Create note array
-    note_array = np.array([(p, float(on) / time_div,
-                            (off - on) / time_div,
-                            np.round(np.mean(vel)))
-                           for p, on, off, vel in notes],
-                          dtype=[('pitch', 'i4'),
-                                 (f'onset_{time_unit}', 'f4'),
-                                 (f'duration_{time_unit}', 'f4'),
-                                 ('velocity', 'i4')])
+    note_array = np.array(
+        [
+            (p, float(on) / time_div, (off - on) / time_div, np.round(np.mean(vel)))
+            for p, on, off, vel in notes
+        ],
+        dtype=[
+            ("pitch", "i4"),
+            (f"onset_{time_unit}", "f4"),
+            (f"duration_{time_unit}", "f4"),
+            ("velocity", "i4"),
+        ],
+    )
     return note_array
 
 
-def match_note_arrays(input_note_array, target_note_array,
-                      fields=None, epsilon=0.01,
-                      first_note_at_zero=False,
-                      check_duration=True,
-                      return_note_idxs=False):
-    """ Compute a greedy matching of the notes of two note_arrays based on
+def match_note_arrays(
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.